mewarpx/tests/test_checkpoint_and_restart.py

```diff
@@ -72,10 +72,52 @@ def test_create_checkpoints():
 
     run = get_run()
 
-    checkpoint = CheckPointDiagnostic(DIAG_STEPS, CHECKPOINT_NAME)
+    checkpoint = CheckPointDiagnostic(
```

Comment:
For the sake of test coverage, we should make one of the tests have `clear_old_checkpoints=True`.